sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -73,6 +73,10 @@ class JsonFunctionsSuite extends QueryTest with SharedSQLContext {
     checkAnswer(
       df.select($"key", functions.json_tuple($"jstring", "f1", "f2", "f3", "f4", "f5")),
       expected)
+
+    checkAnswer(
+      df.selectExpr("key", "json_tuple(jstring, 'f1', 'f2', 'f3', 'f4', 'f5')"),
+      expected)
   }
 
   test("json_tuple filter and group") {
```

Comment:
@yhuai I found two test cases that i am going to link it for you to look at. One is in lateral view and the other shows regular usage. Both these include aliases of the form ci. Please take a look.

Regular usage 
https://github.com/apache/spark/blob/149cd692ee2e127d79386fd8e584f4f70a2906ba/sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala#L78-#L84

Lateral view
https://github.com/apache/spark/blob/d83c2f9f0b08d6d5d369d9fae04cdb15448e7f0d/sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala#L1452-#L1466